File: yeseung/1주차/PG_42748_k번째수.cpp
#include <string>
#include <vector>
#include<algorithm>

using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> answer;

    for(int index = 0;index<commands.size(); ++index){
        int i = commands[index][0];
        int j = commands[index][1];
        int k = commands[index][2];
        
        vector<int> tmp(array.begin()+i-1, array.begin()+j);
        sort(tmp.begin(),tmp.end());
        
        answer.push_back(tmp[k-1]);
    }
    return answer;
}

File: kitttae/1주차/PG_42748_K번째수.java
import java.util.*;
class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        
        for (int len = 0; len < commands.length; len++) {
            int i = commands[len][0];
            int j = commands[len][1];
            int k = commands[len][2];
            
            int[] arr = Arrays.copyOfRange(array, i - 1, j);
            Arrays.sort(arr);
            answer[len] = arr[k-1];
        }
        
        return answer;
    }
}
